src/reference/asciidoc/whats-new.adoc

```diff
@@ -51,15 +51,21 @@ Starting with _version 5.0.5_, generated bean names for the components in an `In
 
 See <<java-dsl-flows>> for more information.
 
-==== AMQP Changes
+==== @Publisher annotation changes
+
+Starting with _version 5.1, the `@Publisher` AOP functionality has to be turned on explicitly via `@EnablePublisher` or via `<int:enable-publisher>` sub-element on the `<int:annotation-config>`.
+
+See <<publisher-annotation>> for more information.
+
+=== AMQP Changes
 
 `ID` and `Timestamp` header mapping changes in the `DefaultAmqpHeaderMapper`.
 See the note near the bottom of <<amqp-message-headers>> for more information.
 
 The `contentType` header is no longer incorrectly mapped as an entry in the general headers map.
 See <<amqp-content-type>> for more information.
 
-==== JDBC Changes
+=== JDBC Changes
```

Comment:
I don't think these should be reduced to 3x `=` - that is for top-level "New Components" and "General Changes" and there is one more `==== Aggregator Changes` afterwards.

I can change them back during the merge if you agree.

Reply:
Oh, I see; no you are right; will fix when rebasing.

Reply:
Ah! I see what you mean. 
Well, the `General Changes` name is confusing though...
May be change it to the ``Other Changes` since it goes already after `New Components`  :smile:

Otherwise yes: please, revert it back during merge if it is only concern.

Reply:
Ah, right! I definitely meant there `general` as for all the core changes and each other modules have their own top level paragraph.